lib/req/steps.ex

```diff
@@ -197,6 +201,14 @@ defmodule Req.Steps do
     end
   end
 
+  def load_netrc(request, :env) do
+    path =
+      System.get_env("NETRC") ||
+        raise "could not find .netrc path, please set the NETRC environment variable"
```

Comment:
This behavior is meant to be consistent with what happens if `System.user_home!/1` is not set.